src/website/views/components/avatar.pug

```diff
@@ -0,0 +1,323 @@
+      <div class="a-avatar -md">AA</div>
+      <div class="a-avatar -lg">AA</div>
+      <div class="a-avatar -xl">AA</div>
+      <div class="a-avatar -xxl">AA</div>
+
+  #sizedIcons.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar a-icon -${size}`)
+              svg
+                use(xlink:href=`#icon-user`)
+            p.-text= size
+
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm2">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm3">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -md">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -lg">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xxl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+  #sizedImages.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar -${size}`)
+              img(src='../../assets/images/avatar.jpg')(alt="avatar")
+            p.-text= size
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm2">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm3">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -md">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -lg">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -xl">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -xxl">
+        <img src="path/to/image.jpg">
+      </div>
+
+h3 Labeled avatars
+
+p.-text
+  | Avatars can be accompanied by a label. Include the avatar and the label in a div with a <code>a-avatarGroup</code>
+  | class. Labeled avatars still support type, size and color variations.
+.example.-mb3
+  .-p3.a-grid
+    each color in colors
+      .-mb1.a-col.-w3
```

Comment:
We need to add responsive grid classes to the examples.
For labeled avatars, we would change `-w3` to:  `-w6 -sm--w4 -lg--w3`

So by default it renders in 2 columns, then 3 on small and medium devices, then 4 on large devices